ts/DataGrid/DataGrid2/DataGridOptions.d.ts

```diff
@@ -156,6 +156,22 @@ export interface ColumnOptions {
      * The format of the column header.
      */
     headFormat?: string;
+
+    /**
+     * Extendable method for formatting each cell's in DataGrid.
```

Comment:
I don't think it is `each cell's in DataGrid.` It should be `within the given column.`

Reply:
@vazonik ?